build/task-build-browser.js

```diff
@@ -89,11 +89,18 @@ export async function buildFile(sourceFile, sourceStats, args = []) {
 
 export default async function babelBuild(args = []) {
   const source = path.resolve(path.join(__dirname, '..', 'app'));
-  const paths = await fs.walk(source);
+  let paths = await fs.walk(source);
+
+  // Exclude the `ui/content` dir. This is super ugly here, but we won't
+  // need any of this after we switch our entire build system to webpack.
```

Comment:
We also have glob as a dependency already, something like glob("app/**" { ignore: "app/ui/content/**"}) would probably do the trick.